src/views/articleDetailView/index.js

```diff
@@ -52,18 +64,19 @@ export class ArticleDetailView extends Component {
     if (article.title === "") {
       return (
         <div className="h-100 w-100 p-5 m-5  text-center">
-          {toast.error("Article not found", {
+          {/* {toast.error("Article not found", {
```

Comment:
can you please remove this commented code?

Reply:
It is removed

Reply:
I have fixed that @BagzieGracious